src/configuration/configuration_controller.py

```diff
@@ -0,0 +1,88 @@
+from plumbum import local, FG, BG, TF, RETCODE, ProcessExecutionError
+import subprocess
+
+
+class ConfiguartionController:
+    def __init__(self):
+        self._multichain_util = local['multichain-util']
+        self._multichain_d = 'multichaind'
+        self._multichain_daemon = "-daemon"
+        self._create = self._multichain_util['create']
+        self._multichain_path = "~/.multichain/"
```

Comment:
For interoperability, please use Python’s in-built `os` API. Right now, running this code on DOS machines will throw an error because Windows does not define the `HOME_PATH` as `~`, but `%USERPROFILE%` (or something like that)